docs/rst/tutorials/core/deployment/ds_locators_runtime/ds_locators_runtime.rst

```diff
@@ -0,0 +1,369 @@
+publications.
+
+Prerequisites
+-------------
+
+First of all, make sure that Vulcanexus Humble is installed.
+The docker installation is required for this tutorial (see :ref:`Docker installation <docker_installation>`).
+
+.. note::
+
+    It is highly recommended to complete the :ref:`Configuring Fast-DDS QoS via XML profiles <tutorials_xml_profiles_intro>`
+    tutorial to learn how to configure ROS 2 via XML configuration files.
+
+
+Set up the discovery environments
+---------------------------------
+
+Open five terminals, and run the Vulcanexus Humble image in each one with the following commands:
+
+.. tabs::
+
+    .. tab:: Discovery Environment 0
+
+        .. tabs::
+
+            .. tab:: Server_0
+
+                .. code-block:: bash
+
+                    docker run -it --name server_0 ubuntu-vulcanexus:humble-desktop
+
+            .. tab:: Client_talker_0
+
+                .. code-block:: bash
+
+                    docker run -it --name client_pub_0 ubuntu-vulcanexus:humble-desktop
+
+            .. tab:: Client_listener
+
+                .. code-block:: bash
+
+                    docker run -it --name client_sub ubuntu-vulcanexus:humble-desktop
+
+    .. tab:: Discovery Environment 1
+
+        .. tabs::
+
+            .. tab:: Server_1
+
+                .. code-block:: bash
+
+                    docker run -it --name server_1 ubuntu-vulcanexus:humble-desktop
+
+            .. tab:: Client_talker_1
+
+                .. code-block:: bash
+
+                    docker run -it --name client_pub_1 ubuntu-vulcanexus:humble-desktop
+
+
+
+Configure the discovery entities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The *servers* configuration is really simple, the *server* ``discovery id`` would be set as ``0`` in the first discovery environment, and ``1`` in the other one.
+
+.. tabs::
+
+    .. tab:: Server 0
+
+        .. code-block:: bash
+
+            fastdds discovery --server-id 0
+
+    .. tab:: Server 1
+
+        .. code-block:: bash
+
+            fastdds discovery --server-id 1
+
+The outputs should look similar to the followings:
+
+.. tabs::
+
+    .. tab:: Server 0
+
+        .. code-block:: bash
+
+            ### Server is running ###
+            Participant Type:   SERVER
+            Server ID:          0
+            Server GUID prefix: 44.53.00.5f.45.50.52.4f.53.49.4d.41
+            Server Addresses:   UDPv4:[0.0.0.0]:11811
+
+    .. tab:: Server 1
+
+        .. code-block:: bash
+
+            ### Server is running ###
+            Participant Type:   SERVER
+            Server ID:          1
+            Server GUID prefix: 44.53.01.5f.45.50.52.4f.53.49.4d.41
+            Server Addresses:   UDPv4:[0.0.0.0]:11811
+
+The output displays the ``server id`` set, followed by the server GUID prefix.
+Server addresses ``0.0.0.0`` net mask allows all addresses to be set, but we need each server specific address.
```

Comment:
It's more that 0.0.0.0 tells Fast DDS to listen on all available interfaces. Also, the second part of the sentence is not clear.

Furthermore, please refrain from using informal/direct speech. We'd rather use reported speech for technical documentation